base_location_geonames_import/views/res_country.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record model="ir.ui.view" id="view_country_geonames_form">
+        <field name="name">res.country.geonames.form</field>
+        <field name="model">res.country</field>
+        <field name="inherit_id" ref="base.view_country_form"/>
+        <field name="type">form</field>
+        <field name="arch" type="xml">
+            <field name="code" position="after">
+                <field name="geonames_state_name_column" />
+                <field name="geonames_state_code_column" />
+            </field>
+        </field>
+    </record>
+
+    <menuitem

```

Comment:
Why adding an extra menu? Leave the current one.

Reply:
Is to allow people to access every country item and "manually" modify the two field to fix the import of states for that country

Reply:
This menu should go away, as the maintenance should be done in their natural place, not here.

Reply:
Ok that's reasonable, I remove it